client/gutenberg/extensions/related-posts/edit.jsx

```diff
@@ -4,6 +4,7 @@
  * External dependencies
  */
 import classNames from 'classnames';
+import moment from 'moment';
```

Comment:
Boom, wonder what happens to bundle size due to this. 🤔  [`@wordpress/date`](https://github.com/WordPress/gutenberg/tree/master/packages/date) would be available globally already, would that work here?

Reply:
Great catch! Done in https://github.com/Automattic/wp-calypso/pull/26530/commits/5cea80ad1d5b307c4a7809f3ca30e8733f94cf26 

The difference is huge! 

`moment`:
![](https://cldup.com/89RDXCPfsA.png)

`@wordpress/date`:
![](https://cldup.com/l27MaVlHYA.png)

We only need to require `@wordpress/date` in Jetpack too. Here's a PR for that: https://github.com/Automattic/jetpack/pull/10173